migrations/versions/600_change_numberofsuppliers_to_integer.py

```diff
@@ -0,0 +1,61 @@
+"""Change numberOfSuppliers to integer
+
+Revision ID: 600
+Revises: 590
+Create Date: 2016-05-06 11:37:59.204714
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '600'
+down_revision = '590'
+
+import re
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+from sqlalchemy.dialects import postgresql
+
+briefs = table(
+    'briefs',
+    column('id', sa.Integer),
+    column('data', postgresql.JSON),
+)
+
+
+def upgrade():
+    conn = op.get_bind()
+    for brief in conn.execute(briefs.select()):
+        # Skip briefs with missing or integer 'numberOfSuppliers'
+        if brief.data.get('numberOfSuppliers') is None or isinstance(brief.data['numberOfSuppliers'], int):
+            continue
+
+        # Get the last number in the 'numberOfSuppliers' string
+        number_of_suppliers = re.search(r'(\d+)(?!.*\d)', brief.data['numberOfSuppliers'])
```

Comment:
Yeah, that's based and tested on the values we have in production right now. I'm going to check that it's still fine right before merging and releasing, can't think of any other way to do it.